support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -167,6 +181,17 @@ export function SupporterPlusLandingPage({
 									<PaymentFailureMessage />
 									<DirectDebitContainer />
 								</ContributionsStripe>
+								<PaymentTsAndCs
+									countryGroupId={countryGroupId}
+									contributionType={contributionType}
+									currency={currencyId}
+									amount={getAmount(
+										selectedAmounts,
+										otherAmounts,
+										contributionType,
+									)}
```

Comment:
Could use the `getUserSelectedAmount` selector instead? Although it can return `NaN` if the user's chosen 'other amount' but not put anything in yet.

Reply:
I've passed through the amount from `getUserSelectedAmount` and updated the `amountCopy` variable that uses it in the component to account for the amount potentially being NaN